linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_fdisk/bash/shared.sh

```diff
@@ -1,5 +1,11 @@
 # platform = multi_platform_ubuntu
 
+{{%- if product in ['ubuntu2004'] %}}
```

Comment:
please drop this block

Reply:
Relating to the couple comments below for this rule, I agree that `/usr/sbin` -> `/sbin`, this does work fine but the OVAL defs for DISA are expecting `/usr/bin`. I'm going to push my modifications removing these blocks, but just a heads up.